packages/microcosm-www-next/src/components/button.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { BrowserIcon, PlanetIcon } from '../images'
+
+export default class Button extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {runFadeAnim: ''}
+  }
```

Comment:
I'm not sure babel-stage-2 is setup on this project, but we really like to use class instance properties when we can:

```
class Button extends React.Component {
  state = { runFadeAnim: '' }
}
```

This is functionally equivalent to what you have (the assignment happens after the constructor runs). What do you think?

Reply:
sounds good, it worked on my machine!